packages/react-core/src/components/TreeView/TreeViewRoot.tsx

```diff
@@ -9,7 +9,7 @@ export interface TreeViewRootProps {
   /** Child nodes of the tree view */
   children: React.ReactNode;
   /** Flag indicating if the tree view has checkboxes. */
-  hasChecks?: boolean;
+  hasCheckboxs?: boolean;
```

Comment:
Similar to a previous comment, instances of `hasCheckboxs` in this file just need to be updated:

```suggestion
  hasCheckboxes?: boolean;
```